docs/TROUBLESHOOTING.md

```diff
@@ -74,9 +74,9 @@ $ oc adm node-logs -l kubernetes.io/os=windows --path=/containerd/containerd.log
 ## How to collect Windows application event logs
 
 The Get-WinEvent shim on the kubelet logs endpoint can be used to collect application event logs from Windows machines.
-E.g. for the docker runtime service:
+E.g. for the containerd runtime service:
 ```shell script
-$ oc adm node-logs -l kubernetes.io/os=windows --path=journal -u docker
+$ oc adm node-logs -l kubernetes.io/os=windows --path=/containerd/containerd.log
 ```
```

Comment:
We can remove this, it is a repeat of the `## How to collect containerd runtime logs` section above

Reply:
I agree with @saifshaikh48 this whole section can be removed, no need for example here as we are already giving an example above this step both for containerd and kubelet logs. Moreover this is not collecting win-event logs now.

Reply:
This section is still useful and valid, we just need to find a good example that fit 
```
oc adm node-logs -l kubernetes.io/os=windows --path=journal -u <LOG_NAME>
```

Reply:
I will use the above template line instead of containerd example